### ISSUE TITLE
Gifted Ultra Rare Collectors Cache not tradable

### ISSUE THREAD

**Issue by uaaiuoioo** [ValveSoftware/Dota2-Gameplay#11414]
### Cosmetic Item Name

Radiant Conqueror

### Description

Item got bug in my inventory after I unbox a gifted Radiant Conqueror bundle.  It's not tradable and marketable (Look pictures). At first, after opening it like a year ago, a trade cooldown message pop off but the tags at the bottom is still not tradable and marketable. Earlier today I remove the gifter information just to see if something gonna happen but no changes.

### Example Match ID

None

### Screenshots

![IMG_1152](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143911615/b5f8f9ce-2271-47f7-83da-3a312ca80990)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143911615/85018914-efa8-41ce-b164-cecb5790ee11)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143911615/3e275602-687b-4638-879e-c93239995a01)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143911615/f976d907-8ff3-4239-a1a6-6c45732302cf)

**Comment by shaileshgarg** [comment]
The set bundled/un-bundled should be marketplaceble since it is a Ultra Rare. don't know if gifting effects the marketability. @Aluerie

**Comment by uaaiuoioo** [comment]
> The set bundled/un-bundled should be marketplaceble since it is a Ultra Rare. don't know if gifting effects the marketability. @Aluerie

Yeah I don't know what happen so I assume it's a bug because I got it via gift when it's not yet available in the market.
